Jimmy-D-submissions/src/main code

```diff
@@ -0,0 +1,347 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+int main() {
+    int move_b, move_f, steps, count, num;
+    char code, letter;
+    count=0;
+    cout<<"Do you want to Decode or Create a code? Type D to Decode and C to create a code.\n";
+    cin>>code;
+    if (code='D'){
+        cout<<"You have chosen to Decode. How many steps would you like to move forward?(If you want to move backwards use a negative sign up to -25)\n";
+        cin>>steps;
+        cout<<"What is the code you would like to Decode?(lowercase only)\n";
+        while(count!=999){
+            cin>>letter;
+            if (letter=='a'){
+                num=1;
+            }
```

Comment:
These `if` statements are not the "right" way to do this!

Much better is to use a dictionary defined outside the function:

```cpp
map<int, string> letter_to_number;
letter_to_number[a] = 1;
...
letter_to_number[z] = 26;
```